### PR TITLE
Reps reset

### PR DESCRIPTION
**Pull request by bcolloran**
@hamilton for discussion -- this just resets the reps to use react inspector in all cases save for errors

(ps i know the tests are failing, this is just for you to see what it feels like if we start from that baseline. IMO, even tho these reps are not 100% amazing, this moves the ball forward)

### CODE REVIEW

**Comment by mdboom** [comment]
We should try to retain at least the `renderMethod` handling -- this is used by Pandas and Julia to display custom HTML reps.  Even if we don't handle those embedded inside React inspector reps, it would be really handy (for not too much pain) to support them at the top level.

**Comment by bcolloran** [comment]
oh good point, thanks @mdboom !

**Comment by bcolloran** [comment]
@mdboom -- looking at this issue again and looking back at https://github.com/iodide-project/iodide-examples/blob/master/output-handling.html#L49
I've had the following thoughts/questions about the iodideRender functionality--

1) I think we should restrict it to HTML strings only. If we move our console to a separate iframe with remote reps a la dev tools, passing real DOM objs around won't be possible
2) I think maybe we should only support the iodideRender() approach. in JS users don't have to change their implementation of a class, they can monkeypatch the prototype, which should be fine for something like adding a render function, and I kind of wonder if also supporting the `iodide.addOutputHandler` approach adds more complexity and brittleness for small gains in increased functionality

**Comment by bcolloran** [comment]
good points, thanks @mdboom.

regarding interactive plotting, i don't think we want plots remote from the data, just console reps. wrt plots, i think the path forward is to continue to do basically what we do with e.g. matplotlib plots right now, which is to render them directly into the report. In all scenarios that @hamilton  and I have discussed, the eval frame scope will continue to have direct access to the report DOM, so full interactivity will not be an issue. If we decide that this is a bad UX for some reason we want to be able to support interactive plots that are not displayed inline in the report, we could consider a very minimal "plot pane" within the eval frame, which could be quite a lot slimmer than the full machinery of the console. FWIW, in my dogfooding experience displaying inline in the report during the exploratory phase of has worked fine.

(Again, the long term architectural motivation here is that we want to get the eval frame as close as we can to being a very dumb iframe with as few parts of our app as possible. This will both make it harder for users to break the app, make it easier for them to do what they want with their css without having to navigate around the extra styles we have in place to support the app UI, and simplify the interaction between the parts of the app that run untrusted code and our own app logic)

re `iodide.addOutputHandler`, your points are well taken -- agree that keeping that in but only at the top level seems workable, so I'll make sure to have that in this PR.

File: src/components/reps/__tests__/error-handler.test.js [hunk 1]
```diff
@@ -1,166 +1,176 @@
...
-    /* eslint-disable */
     const mockFrames = [
       {
-        "isEval": true,
-        "columnNumber": 7,
-        "lineNumber": 1,
-        "fileName": "cell",
-        "functionName": "eval",
-        "source": "@http://localhost:8000/iodide.eval-frame.dev.js line 103208 > eval:1:7"
+        isEval: true,
```
Comment (bcolloran) [inline src/components/reps/__tests__/error-handler.test.js]: all these changes in these objects are Prettier format things

**Comment by bcolloran** [comment]
I've pushed another revision for discussion. 

Compare this:
https://extremely-alpha.iodide.io/notebooks/219/
to this for local testing (a few changes are needed to work locally b/c of the naming convention changes i mention above)
https://gist.github.com/bcolloran/f2dd808018ff007acd8d5254e323508d

Overall I think this is an incremental improvement. Our reps have kind of rotted, and react-inspector, while not uniformly better, is at least consistent, mostly working, and allows you to pretty much fully inspect your objects, without quite so many of the problems we never got around to solving with truncation etc. This is not the end state, but I think it does move the ball forward. I'd love to land this this week.

**Comment by mdboom** [comment]
> regarding interactive plotting...

Thanks for the clarification.  I think we're in agreement there.  Real direct access to the DOM is a differentiating feature for us that we should maintain.

Regarding terminology -- I like `render`, but agree it has Reactish baggage and our signature for `render` will be different from React's.  So I think that brings me to `reps` as my first choice.

**Comment by bcolloran** [comment]
Now with changelog and docs! New habits FTW! :-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # 0.0.4 (Unreleased; add upcoming change notes here)
```
[the remark this answers is not in the document]
Comment (hamilton) [reply CHANGELOG.md]: that's fine, but let's go with 0.1.0 and be fine with breaking changes happening while we're alpha so as not to confuse new folk

**Pull request merged**